.drone.yml

```diff
@@ -9,6 +9,27 @@ clone:
     tags: true
 
 pipeline:
+  download_translations:
+    image: jonasfranz/crowdin
+    pull: true
+    secrets: [ crowdin_key ]
```

Comment:
This needs to be added to drone server right?

Reply:
No this is already added during the upload process